dockers/docker-snmp-sv2/snmpd.conf.j2

```diff
@@ -61,7 +61,7 @@ sysServices    72
 #
 #  Process Monitoring
 #
-# TODO: should we enable snmp based monitoring of sswsyncd and other processes?
+# todo: should we enable snmp based monitoring of sswsyncd and other processes?
```

Comment:
Why change TODO to lowercase? All other TODOs in the file are uppercase.

Reply:
It's in a revert commit. Since it is a comment format issue, I will not refix it again.